### PR TITLE
Buffs Traitor Carp

### PR DESCRIPTION
**Pull request by Iamgoofball**
tried this too, it died instantly and didn't move fast enough

### CODE REVIEW

**Comment by kevinz000** [comment]
please no I got instacritted by 20 of these once.

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 1]
```diff
@@ -66,11 +66,28 @@
...
 
+/mob/living/simple_animal/hostile/carp/traitor
+	icon = 'icons/mob/alienqueen.dmi'
+	name = "Weaponized Attack Carp"
+	desc = "Designed by the Syndicate for maximum murder."
+	icon_state = "megacarp"
+	icon_living = "megacarp"
+	icon_dead = "megacarp_dead"
+	icon_gib = "megacarp_gib"
+	maxHealth = 75
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: ayy

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 2]
```diff
@@ -66,11 +66,28 @@
...
+	health = 75
+	pixel_x = -16
+	mob_size = MOB_SIZE_LARGE
+
+	melee_damage_lower = 20
+	melee_damage_upper = 30
```
Comment (KorPhaeron) [inline code/modules/mob/living/simple_animal/hostile/carp.dm]: lmao

**Comment by Iamgoofball** [comment]
Why

**Comment by PKPenguin321** [comment]
imagine 20 of these. 20 walking eswords

**Pull request closed without merging**